CMake/kwiver-depends-SVM.cmake

```diff
@@ -0,0 +1,12 @@
+# Required SVM external dependency
+find_library(SVMLIB svm)
```

Comment:
Why do you have it labeled svmlib instead of libsvm? (generally it's best to have it labeled as the exact library name to avoid confusion)

You might want to follow the paradigm of the other depends so it's consisitent, e.g. add the KWIVER_ENABLE_SVM option here

Reply:
I tried to incorporate the changes you suggested.